src/components/__tests__/EmptyThought.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import App from '../../App'
+
+// eslint-disable-next-line fp/no-let
+let wrapper = null
+
+beforeAll(async () => {
+  wrapper = await window.createTestApp(<App />)
+})
+
+afterAll(() => {
+  window.cleanupTestApp()
+  wrapper = null
+})
+
+it('create and edit empty thought', async () => {
+  const thoughts = wrapper.find('div.transformContain div ul.children')
+  const thought = thoughts.find(
+    'li.leaf div.thought-container div.thought div.editable',
```

Comment:
Omit tag names, i.e. `.leaf .thought-container .thought .editable`. In this case I think we can simply select `.editable`.

Reply:
I tried it, but tests fail that way